### DISCUSSION TITLE
Using the defer loading inside a foreach

### DISCUSSION THREAD

**Discussion by Epitaffio** [livewire/livewire#4975] [1. Help] [answered]
Hey there, this is a copy of a question I asked on [r/laravel](https://www.reddit.com/r/laravel/comments/uscd67/livewire_defer_and_foreach/).

The gist is:
I have a full page livewire component, inside the component I have a foreach loop.

Inside this loop I have a method call in the component that return some data.

Every call to the method can take different time based on a lot of things, so I would love to update the rows indipendently as soon as they have the result.

As per what I have tried, I have tried to use the defer loading, but this way the result came all in one ajax call (more detail on my implementation on the reddit link).

There is a way to do it in a "livewire way"? Thanks in advance!

**Comment by joshhanley** [comment, marked as answer]
@Epitaffio only way for them to update in their own network requests would be to use nested components, so creating a row component, that deals with updating that row.

Hope this helps!
